RELEASING.md

```diff
@@ -154,6 +154,7 @@ JDK_FIX_VERSION=0
    * `targetConfigurations`: remove all "hotspot" entries
    * `releaseType`: `Nightly`
    * `overridePublishName`: github binaries publish name, e.g. `jdk8u232-b09_openj9-0.17.0-m1` or `jdk-11.0.5+10_openj9-0.17.0-m1`
+   (Note: Everything before the underscore should be copied from the OPENJDK\_TAG value inside <extensions_repo_url>/closed/openjdk-tag.gmk)
```

Comment:
Why the backslash in `OPENJDK\_TAG` (I don't see other underscores escaped)?

Reply:
In markdown documents, anything after an underscore is displayed in _italics_. I didn't want italics, but I did want an underscore.

Reply:
Backslash removed.